front/pages/w/[wId]/builder/assistants/index.tsx

```diff
@@ -135,42 +97,27 @@ export const getServerSideProps = withDefaultUserAuthRequirements<{
     };
   }
 
-  // TODO(agent-discovery) Remove feature-flag
-  const featureFlags = await getFeatureFlags(owner);
-  const hasAgentDiscovery = featureFlags.includes("agent_discovery");
-
-  if (!auth.isBuilder() && !hasAgentDiscovery) {
```

Comment:
Cause now any user can access this page right?

Reply:
yes, everybody has access (`!hasAgentDiscovery` is always false)